internal/repos/syncer.go

```diff
@@ -547,6 +547,8 @@ func (s *Syncer) syncRepo(ctx context.Context, store *Store, insertOnly bool, pu
 	return diff, nil
 }
 
+// upserts returns a slice of only modified and added repos from diff. Deleted
```

Comment:
i found the original version a little hard to read, how does this sound?
```suggestion
// upserts returns a slice containing modified or added repos from a Diff. Deleted
```